examples/basic-demo-tableload.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates basic usage of the firefly_client API show_table for the file with multiple tables.\n",
+    "\n",
+    "Note that it may be necessary to wait for some cells (like those displaying an table) to complete before executing later cells."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for Python 2/3 compatibility"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division, absolute_import"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for firefly_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from firefly_client import FireflyClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we use the irsaviewer application for the server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "host='127.0.0.1:8080'\n",
+    "channel = 'myChannel8'\n",
+    "html = 'firefly.html'\n",
+    "\n",
+    "fc = FireflyClient(host, channel=channel, html_file=html)\n",
+    "fc.launch_browser()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load a FITS containing multiple tables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "localfile = '/hydra/cm/firefly_test_data/FileUpload-samples/fits/problemFits/src.fits';\n",
```

Comment:
Let's tell the user that the files in this notebook are available in the git-lfs repo at https://github.com/lsst/firefly_test_data, with some text in a Markdown cell preceding this code cell.

A user will not have the /hydra developer setup. To run this test notebook, I modified this cell like this:
```
import os
testdata_repo_path = '/Users/shupe/data/lsst/'
localfile = os.path.join(testdata_repo_path,'firefly_test_data/FileUpload-samples/fits/problemFits/src.fits')
```